### PR TITLE
Use ASDF for installing Erlang/OTP

### PR DESCRIPTION
**Pull request by 0rca**
This addresses #160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,39 @@ The Hamler 0.1 compiler was forked from [PureScript][PureScriptHamler] 0.13.6. T
...
+1. Install asdf-vm [Erlang plugin](https://github.com/asdf-vm/asdf-erlang)
+
+  ```shell
+  asdf plugin-add erlang
+  ```
+
+2. Install Erlang
+
+  ```shell
+  asdf install
```
Comment (zhanghongtong) [inline README.md:98]: I think clone hamler from the git repo should be executed before `asdf install`, because `asdf install` requires `.tool-versions`

**Review by Rory-Z — COMMENTED** [review]
Hi, @0rca 
Thank you for your PR. I put forward a suggestion and hope to get your reply

**Comment by zhanghongtong** [comment]
Hi, @emqplus 
Do you think we should guide the user how to install erlang in the readme? Or we just tell the user the required version of erlang, it is up to the user to decide how to install erlang ?

File: README.md (hunk 1 above)
Comment (0rca) [reply README.md]: Oh, yes, you're right. I've updated the README accordingly.

File: README.md [hunk 2]
```diff
@@ -82,6 +82,41 @@ The Hamler 0.1 compiler was forked from [PureScript][PureScriptHamler] 0.13.6. T
...
+
+0. Install [asdf-vm](https://asdf-vm.com/#/core-manage-asdf-vm?id=install)
+
+1. Install asdf-vm [Erlang plugin](https://github.com/asdf-vm/asdf-erlang)
+
+  ```shell
+  asdf plugin-add erlang
+  ```
+
+3. Clone hamler from the git repo and change directory
```
Comment (zhanghongtong) [inline README.md:95]: I think the sequence number of the ordered list should be correct. Although the sequence number rendered by markdown is correct, we should also keep the text content correct

**Review by Rory-Z — COMMENTED** [review]
Hi, @0rca 
I think the sequence number of the ordered list should be correct. Although the sequence number rendered by markdown is correct, we should also keep the text content correct

File: README.md (hunk 2 above)
Comment (0rca) [reply README.md:95]: Right. I've addressed this as well.

Comment (zhanghongtong) [reply README.md:95]: Thx :)

**Review by Rory-Z — APPROVED** [review]

**Pull request closed without merging**